### PR TITLE
mkck: copy zypper repo artifacts for mkck

### PR DESCRIPTION
**Pull request by kshtsk**
jenkins-jobs update jjb/mkck/mkck-trigger.yaml mkck-sles-ses6

Signed-off-by: Kyr Shatskyy <kyrylo.shatskyy@gmail.com>

### CODE REVIEW

File: jjb/mkck/mkck-trigger.yaml [hunk 1]
```diff
@@ -506,6 +515,37 @@
...
+            name: Deliver artifacts
+            #condition: ALWAYS
+            condition: SUCCESSFUL
+            projects:
+                - name: storage-deliver-mkck-artifact
+                  current-parameters: true
+                  abort-all-job: true
+                  predefined-parameters: |
+                    DIST_CEPH={dist_ceph}
+                    SITE=ovh.net
```
Comment (toabctl) [inline jjb/mkck/mkck-trigger.yaml:528]: should `SITE` be configurable to be able to run it on ECP?

File: jjb/mkck/mkck-trigger.yaml [hunk 2]
```diff
@@ -514,9 +554,18 @@
...
             TARGET_NAME=$name
             EOF
-
         - inject:
             properties-file: $TARGET_FILE
+        - shell: |
+            . v/bin/activate
+            . sesci/common/util
+            for i in artifacts*.yaml ; do
+                # now jq black magic
```
Comment (toabctl) [inline jjb/mkck/mkck-trigger.yaml:563]: hm. can't we just use code from `common/sesci` or merge the `artifacts2zypper.py` script from https://gitlab.suse.de/ceph/storage-ci/-/merge_requests/110/diffs?commit_id=b54db153166839faaaee40d8f93de39c5a667eb7#diff-content-b3d020c337ec05b679cafccd1f7a8ed2cb755cbe and use that?

Comment (kshtsk) [reply jjb/mkck/mkck-trigger.yaml:563]: I don't see any benefit in artifacts2zypper.py 38 lines of pythonism, in comparison to one liner jq command. I mean it does not worth to create a separate python script which can be done with single command. As for common/sesci it is a separate repo too.
Btw, as for artifacts2zypper.py, maybe there is a sense to add format key for artifacts.py to jenerate zypper config file or zypper command.

File: jjb/mkck/mkck-trigger.yaml (hunk 1 above)
Comment (kshtsk) [reply jjb/mkck/mkck-trigger.yaml:528]: Yes, it should be configurable but I don't remember exactly that copy-artifacts could render a placeholder with parameter for predefined-parameters, thus, I just left it here, we can create another patch to do this.

Comment (toabctl) [reply jjb/mkck/mkck-trigger.yaml:528]: ok

File: jjb/mkck/mkck-trigger.yaml (hunk 2 above)
Comment (toabctl) [reply jjb/mkck/mkck-trigger.yaml:563]: `artifacts2zypper.py` does some error checking. And one-liners are imho not always better. This one liner is just difficult to get correct...
Anyway, I see you are in favor of using this approach

**Review by toabctl — APPROVED** [review]

**Pull request merged**